### PR TITLE
[ADD][8.0] account_payment_order_sequence

### PR DESCRIPTION
**Pull request by andhit-r**
# 
# Payment Order Sequence

This module add feature to create payment order reference
based on payment mode. Each payment mode could have different
sequence.
# Configuration

To configure specific sequence for payment mode, you need to:
1. Open or create payment mode
2. Choose sequence to be used

Payment mode with empty sequence will use original payment
order sequence

### CODE REVIEW

File: account_payment_order_sequence/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Payment Order Sequence
+======================
+
+This module add feature to create payment order reference
```
Comment (pedrobaeza) [inline account_payment_order_sequence/README.rst]: s/add/adds

File: account_payment_order_sequence/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Configuration
+=============
+
+To configure specific sequence for payment mode, you need to:
+
+1. Open or create payment mode
+2. Choose sequence to be used
+
+Payment mode with empty sequence will use original payment
+order sequence
```
Comment (pedrobaeza) [inline account_payment_order_sequence/README.rst]: dot at the end

File: account_payment_order_sequence/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Usage
+=====
```
Comment (pedrobaeza) [inline account_payment_order_sequence/README.rst:36]: Add simple usage instructions.

**Review by pedrobaeza — APPROVED** [review]
Little nitpickings

**Comment by andhit-r** [comment]
@pedrobaeza thanks for your reviews. I addressed your comments on https://github.com/OCA/account-payment/pull/104/commits/6cbda3eb46cf24aa08898e658b4244894ec1a971.

**Comment by andhit-r** [comment]
Hi @OCA/accounting-maintainers would you kindly review this PR. Many thanks

**Comment by andhit-r** [comment]
@mikevhe18 @azmimr67 please review

**Comment by azmimr67** [comment]
Functionality has been tested on runboard 👍

**Comment by andhit-r** [comment]
Thanks @pedrobaeza

**Pull request merged**